src/rupali/Assignment14/ShopwithScanner.java

```diff
@@ -0,0 +1,116 @@
+package rupali.Assignment14;
+import java.util.Scanner;
+
+public class ShopwithScanner {
+	int stockofMaggie=50;
+	int stockofDosa=43;
+	int stockofOilPouches=39;
+	int stockofPanipuri=43;
+	int stockofmasala=73;
+	
+	void requestedQ(boolean reqType,int maggieq, int dosaQ, int oilQ, int panipuriQ, int masalaQ ){ 
+		if(maggieq==0) 
+			System.out.println("Sorry maggie running out of stock");
+		else 
+			  if(stockofMaggie>=maggieq) {
+				System.out.println("We served your request for maggie");
+				stockofMaggie=stockofMaggie-maggieq;
+			  }
+				else if(reqType==true) {	
```

Comment:
At ine 15 we are allowing customer to purchase partial maggie right?
then why below condition require ?
please let me know if you have any other scenario in mind, let discuss on skype